modules/integration/src/test/scala/scala/cli/integration/InstallAndUninstallCompletionsTests.scala

```diff
@@ -0,0 +1,37 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.util.Properties
+
+class InstallAndUninstallCompletionsTests extends munit.FunSuite {
+  val zshRcFile  = ".zshrc"
+  val bashRcFile = ".bashrc"
+  val testInputs = TestInputs(Seq(os.rel / zshRcFile -> "", os.rel / bashRcFile -> ""))
```

Comment:
imo it'd be good if the test `rc` files you start with weren't empty, so that we can confirm that only the appropriate lines were removed.